xlswriter/xlswriter.php

```diff
@@ -498,7 +498,7 @@ public function __construct($fileHandle)
         /**
          * Wrap
          *
-         * @return Format
+         * @return $this
```

Comment:
Please rollback these changes as they don't make any sense

Reply:
Please take a look at this working example: 

```php
<?php

use Vtiful\Kernel\Excel;
use Vtiful\Kernel\Format;

class Format2 extends Format 
{
        /**
         * @return $this
         */
        public function lall(): self
        {
	    // ...

            return $this;
        }
}

$excel = new \Vtiful\Kernel\Excel([
    'path' => './'
]);

$excelObject = $excel->fileName('tutorial.xlsx');
$excelHandle = $excelObject->getHandle();

// new format object
$format = new Format2($excelHandle);

// set format
$formatResource = $format->lall()->bold();

var_dump($formatResource); // output: object(Format2)#2 (0) { }
```

Reply:
So `static` return type should be used in this case, isn't it? My point is that `$this` isn't a type but `@return` should have a type declaration. Yes, PhpStorm can resolve type based on `$this` but it seems to me that it's better to follow PhpDoc convention.